public/views/answers.html

```diff
@@ -31,7 +43,7 @@
           <h3>How To Play</h3>
         </div>
         <div class="container">
-          <ol id="oh-el" class="game-instructions">
```

Comment:
why did you remove this class

Reply:
I removed it to because I changed the gaming screen style. My changes are not so much.